src/api/v1/db/oracledb/pizzas-dao.js

```diff
@@ -217,4 +248,104 @@ const getPizzaById = async (pizzaId, query) => {
   }
 };
 
-export { getPizzas, getPizzaById };
+/**
+ * Generate queries and bind params for an ingredients relationship resource collection
+ *
+ * @param {object[]} ingredientRelationshipData the `data` key of
+ * `body.data.relationships.ingredients`
+ * @returns {object[]} an object with `queries` and `bindParams` members
+ */
+const getIngredientQueriesAndBindParams = async (ingredientRelationshipData) => {
+  const bindParams = {};
+  const ingredientIds = ingredientRelationshipData
+    .map((ingredient) => ingredient.id);
```

Comment:
that exact approach does not work unfortunately since `ingredientRelationshipData` is an array and not an object. I don't see a way to use lodash methods for this functionality in a way that simplifies the operation being done here but let me know if you're aware of one.

Reply:
Oh I see. In that case, I think you could do `_.map(ingredientRelationshipData, 'id')`